### PR TITLE
Techno-Conglomerate Cleanup

### PR DESCRIPTION
**Pull request by TheDocOct**
Removes the TCG passcard and ESS suit due to the decanonization of the relevant faction.

### CODE REVIEW

File: code/modules/clothing/spacesuits/rig/suits/light.dm [hunk 1]
```diff
@@ -241,61 +241,6 @@
...
-/obj/item/rig/light/offworlder/techno/equipped
-	initial_modules = list(
-		/obj/item/rig_module/device/healthscanner/vitalscanner,
-		/obj/item/rig_module/chem_dispenser/offworlder,
-		/obj/item/rig_module/actuators/combat,
-		/obj/item/rig_module/fabricator/energy_net,
-		/obj/item/rig_module/chem_dispenser/combat,
-		/obj/item/rig_module/mounted/xray
-		)
 /obj/item/rig/light/offworlder/techno/ninja
```
Comment (listerla) [inline code/modules/clothing/spacesuits/rig/suits/light.dm:244]: You'll need to remove or preferably adjust the ninja Techno-Conglomerate suit, I imagine, alongside the removal of its parent items.

Comment (TheDocOct) [reply code/modules/clothing/spacesuits/rig/suits/light.dm]: > You'll need to remove or preferably adjust the ninja Techno-Conglomerate suit, I imagine, alongside the removal of its parent items.

Missed that. Done.

**Comment by TheDocOct** [comment]
!review

**Comment by TheDocOct** [comment]
!bugfix

**Review by alsoandanswer — CHANGES_REQUESTED** [review]
Stuff has not been removed from the .dmis

**Comment by TheDocOct** [comment]
> Stuff has not been removed from the .dmis

Fixed

**Review by SleepyGemmy — APPROVED** [review]

**Comment by KingOfThePing** [comment]
> Stuff has not been removed from the .dmis

They could still be used in another context, renamed, though

**Review by NonQueueingMatt — APPROVED** [review]

**Review by Alberyk — APPROVED** [review]

**Comment by TheDocOct** [comment]
The TCG ninja hardsuit has been readded with name and description tweaks genercizing it as a 'popular frontier hardsuit'.

**Pull request merged**